Documentation/Storage-Configuration/Object-Storage-RGW/cosi.md

```diff
@@ -43,33 +43,9 @@ The driver is created in the same namespace as Rook operator.
 
 ## Admin Operations
 
-### Create a Ceph Object Store User
-
-Create a CephObjectStoreUser to be used by the COSI driver for provisioning buckets.
-
-```yaml
-apiVersion: ceph.rook.io/v1
-kind: CephObjectStoreUser
-metadata:
-  name: cosi
```

Comment:
During upgrades, users who already created this cosi user will have a smooth transition with no manual steps needed, right?

Reply:
@travisn they need to update the information in the `BucketClass` and `BucketAccessClass` since the secret name is different for this user like <objectstore-cosi> and for normal objectstoreuser has a different format

Reply:
@thotz How about if we add the prefix `rook-ceph-object-user-` to the secret so there are no transition steps?